corehq/apps/tzmigration/planning.py

```diff
@@ -0,0 +1,198 @@
+import sqlite3
+import json
+
+from sqlite3 import dbapi2 as sqlite
+from sqlalchemy import create_engine, Column, Integer, ForeignKey, String, \
+    UnicodeText, Text
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.orm.exc import NoResultFound
+
+Base = declarative_base()
+
+
+class PlanningForm(Base):
+    __tablename__ = 'form'
+
+    id = Column(Integer, primary_key=True)
+    uuid = Column(String(50), nullable=False, unique=True)
+    form_json = Column(UnicodeText, nullable=False)
+
+
+class PlanningCase(Base):
+    __tablename__ = 'case'
+
+    id = Column(Integer, primary_key=True)
+    uuid = Column(String(50), nullable=False, unique=True)
+    case_json = Column(UnicodeText, nullable=True)
+    doc_type = Column(String(50), nullable=True)
+
+
+class PlanningCaseAction(Base):
+    __tablename__ = 'case_action'
+
+    id = Column(Integer, primary_key=True)
+    form = Column(Integer, ForeignKey('form.id'), nullable=False)
+    case = Column(Integer, ForeignKey('case.id'), nullable=False)
+    action_json = Column(UnicodeText, nullable=False)
+
+
+class PlanningDiff(Base):
+    __tablename__ = 'diff'
+
+    id = Column(Integer, primary_key=True)
+    kind = Column(String(50), nullable=False)
+    doc_id = Column(String(50), nullable=False)
+    diff_type = Column(String(50), nullable=False)
+    path = Column(Text(), nullable=False)
+    old_value = Column(UnicodeText, nullable=True)
+    new_value = Column(UnicodeText, nullable=True)
+
+
+class PlanningStockReportHelper(Base):
+    __tablename__ = 'stock_report_helper'
+
+    id = Column(Integer, primary_key=True)
+    form = Column(Integer, ForeignKey('form.id'), nullable=False)
+    stock_report_helper_json = Column(UnicodeText, nullable=False)
+
+
+class PlanningDB(object):
+    def __init__(self, db_filepath):
+        self.db_filepath = db_filepath
+        self._connection = None
+        self.engine = create_engine(
+            'sqlite+pysqlite:///{}'.format(db_filepath), module=sqlite)
```

Comment:
Seems like you figured it out, but the point is to have this be a bit more quick and dirty, and flexible for meddling: if something gets messed up in the planning and I want to re-run it, I just delete the sqlite file and start over without having to think about how if effects any other domains I've run the migration on. I'm sure there are other ways, but this seemed simple and easy and silo'd and flexible